multicorn_das/__init__.py

```diff
@@ -665,14 +747,149 @@ def reregister_callback():
                 das_table_name=table_name,
                 das_startup_cost=str(table.startup_cost)
             )
+
             table_definitions.append(
                 TableDefinition(table_name, columns=column_definitions, options=table_options)
             )
             log_to_postgres(f'Processed table {table_name}', DEBUG)
 
-        log_to_postgres(f'Table definitions are {table_definitions}', DEBUG)
         return table_definitions
 
+class DASFunction(DASBase):
```

Comment:
Consider definition a `MulticornFunction` class to inherit from, like `Fdw`. Define it in `multicorn2`, etc.